.pylintrc

```diff
@@ -0,0 +1,588 @@
+[MASTER]
```

Comment:
this file is the default generated from `pylint --generate-rcfile >.pylintrc`.  no special rules here as opposed to the Rally one.  As noted in the PR description, this repo actually fails linting currently so modifications to the ruleset as well as code cleanup will happen in a follow-up PR.